cpp/ql/test/library-tests/ir/ir/ir.cpp

```diff
@@ -988,6 +988,10 @@ int PointerDecay(int a[], int fn(float)) {
   return a[0] + fn(1.0);
 }
 
+int ExprStmt(int x) {
```

Comment:
Please add another test case with multiple statements and some control flow within the `StmtExpr`.